### PR TITLE
alerta: move to all-packages.nix and use buildPythonApplication

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
Getting rid of `python36`.
I cannot find any mention of `import alerta` in the documentation except for plugins (which I assume would be run by alerta).
I did not test whether this actually works. Running `alertad` produces
```
Traceback (most recent call last):
  File "/nix/store/ima4n6kvfhgd4blxmp4fk6fsgv09g6hs-python3.8-Flask-1.1.2/lib/python3.8/site-packages/flask/cli.py", line 556, in list_commands
    rv.update(info.load_app().cli.list_commands(ctx))
  File "/nix/store/ima4n6kvfhgd4blxmp4fk6fsgv09g6hs-python3.8-Flask-1.1.2/lib/python3.8/site-packages/flask/cli.py", line 398, in load_app
    raise NoAppException(
flask.cli.NoAppException: Could not locate a Flask application. You did not provide the "FLASK_APP" environment variable, and a "wsgi.py" or "app.py" module was not found in the current directory.
```
but that's also the case on master.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 112125` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>alerta</li>
<li>alerta-server</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 112125` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>alerta</li>
<li>alerta-server</li>
</ul>
</details>

File: pkgs/servers/monitoring/alerta/client.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  ];
+
+  doCheck = false;
+
+  disabled = python3.pkgs.pythonOlder "3.6";
+
+  meta = with lib; {
+    homepage = "https://alerta.io";
+    description = "Alerta Monitoring System command-line interface";
+    license = licenses.asl20;
```
Comment (SuperSandro2000) [inline pkgs/servers/monitoring/alerta/client.nix:25]: Can you add yourself as a maintainer?

Comment (dotlambda) [reply pkgs/servers/monitoring/alerta/client.nix:25]: no

Comment (dotlambda) [reply pkgs/servers/monitoring/alerta/client.nix:25]: I might fix other peoples' packages if I notice they're broken, but I don't want to be responsible for stuff I don't use.

Comment (SuperSandro2000) [reply pkgs/servers/monitoring/alerta/client.nix:25]: If there is no maintainer for the package we should consider removing it.

Comment (dotlambda) [reply pkgs/servers/monitoring/alerta/client.nix:25]: @svanderburg added it.
There's even a NixOS module.

**Pull request merged**